scripts/setup_haskell.hs

```diff
@@ -164,18 +170,14 @@ setupHaskell = do
                       (Turtle.begins ("#" <|> "user-message" <|> "  ") *>
                        pure "")
                       (Turtle.input "stack.yaml.bak")))
-          versionedHelperDeps <-
-            fmap Turtle.lineToText <$>
-            Turtle.fold
-              (mfilter
-                (filterHelperDeps . Turtle.lineToText)
-                (Turtle.inshell "stack list-dependencies --separator -"
-                  empty))
-              Foldl.list
-          helperDepStackResolver <- stackResolverText $
-                                      hvnHelperBinDir </> "stack.yaml"
-          forM_ versionedHelperDeps $ \dep ->
-            stackInstall helperDepStackResolver dep
+          -- XXX I could not figure out how to keep the ">" sign unescaped in
```

Comment:
Hmm, the template has `>=` and `<` in it already.  Surprised that `>` is causing trouble.  :thinking: